os/dbuild.sh

```diff
@@ -46,6 +46,37 @@ if ! which docker > /dev/null; then
 	exit 1
 fi
 
+KERNEL_DBG_BIN=
+APP1_DBG_BIN=
+APP2_DBG_BIN=
+CMN_DBG_BIN=
+function FIND_DBGBINFILE()
+{
+	source ${CONFIGFILE}
+	BOARDNAME=${CONFIG_ARCH_BOARD}
+	if [[ "${CONFIG_RAW_BINARY}" == "y" ]]; then
+		EXTNAME=".bin"
+	elif [[ "${CONFIG_INTELHEX_BINARY}" == "y" ]]; then
+		EXTNAME=".hex"
+	elif [[ "${CONFIG_MOTOROLA_SREC}" == "y" ]]; then
+		EXTNAME=".srec"
+	else
+		EXTNAME=""
+	fi
+
+	APP1_DBG_BIN="app1_dbg"
+	APP2_DBG_BIN="app2_dbg"
+	CMN_DBG_BIN="common_dbg"
+	if [[ "${CONFIG_ARCH_BOARD}" == "rtl8721csm" ]]; then
+		KERNEL_DBG_BIN="tinyara.axf"
+	elif [[ "${CONFIG_ARCH_BOARD}" == "rtl8720e" ]]; then
+		KERNEL_DBG_BIN="tinyara.axf"
```

Comment:
`tinyara.axf` is set from os/Makefile. And `.axf` is set as `EXTNAME`. So you can keep just oneline with EXTNAME with including Make.defs. Could you check?

Reply:
ok. Done. Thanks.